.gitignore

```diff
@@ -12,6 +12,7 @@ src/**/*.css
 
 # dependencies
 /node_modules
+yarn.lock
```

Comment:
this is a total nit but i would put it under misc since it's not really a dependency... will work the same regardless

Reply:
agree - fixed